src/staff_ghosts.c

```diff
@@ -564,9 +580,10 @@ void staff_ghosts_loop(void) {
         func_80005E6C();
         return;
     }
-    if (!D_80162DF0) {
+    if (!gPauseTriggered) {
         func_80005B18();
         return;
     }
-    D_80162DF8 = 1;
+    // This only gets triggered when the game is paused.
+    gReplayCannotSave = 1;
```

Comment:
Does it not also get triggered above when oob, or lakitu?

Reply:
Ah, I meant that that specific line of code was only triggered when there's a pause. The variable can be set in other places. I changed the comment to be a little clearer.